### PR TITLE
Fix the integration show command

### PR DESCRIPTION
**Pull request by FlorianVeaux**
The `installedVersion` function relies on importing the check and get the version string from the `__about__.py` file. Importing this file also means importing the check external libraries which will fail in a few cases (ibm_db2 for example).

This PR changes the behavior to fetch the version from `pip freeze`

### CODE REVIEW

**Review by trishankatdatadog — DISMISSED** [review]
LGTM

File: cmd/agent/app/integrations.go [hunk 1]
```diff
@@ -663,16 +663,9 @@ func installedVersion(integration string) (*semver.Version, bool, error) {
...
-	if err != nil {
-		return nil, false, fmt.Errorf("Error validating integration name: %s", err)
-	}
-	if !validName {
-		return nil, false, fmt.Errorf("Cannot get installed version of %s: invalid integration name", integration)
-	}
-	pythonCmd := exec.Command(pythonPath, "-c", fmt.Sprintf(integrationVersionScript, integrationName))
-	output, err := pythonCmd.Output()
+
+	freezeCmd := exec.Command(pythonPath, "-mpip", "freeze", "--no-cache-dir")
```
Comment (ofek) [inline cmd/agent/app/integrations.go]: Nit
```suggestion
	freezeCmd := exec.Command(pythonPath, "-m", "pip", "freeze", "--no-cache-dir")
```

**Review by ofek — DISMISSED** [review]

File: releasenotes/notes/Use-pip-freeze-to-get-the-version-of-integrations-0af058b8fb39c548.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
```
Comment (therve) [inline releasenotes/notes/Use-pip-freeze-to-get-the-version-of-integrations-0af058b8fb39c548.yaml]: I think you want to remove those comments.

File: releasenotes/notes/Use-pip-freeze-to-get-the-version-of-integrations-0af058b8fb39c548.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+fixes:
+  - |
+    Use 'pip freeze' to collect the version of the integrations
```
Comment (therve) [inline releasenotes/notes/Use-pip-freeze-to-get-the-version-of-integrations-0af058b8fb39c548.yaml]: `pkg_resources` instead.

**Review by therve — CHANGES_REQUESTED** [review]

File: cmd/agent/app/integrations.go [hunk 4]
```diff
@@ -37,11 +37,8 @@ const (
 		"and coming from a known source. The Agent cannot perform any verification on local wheels."
 	pythonMinorVersionScript = "import sys;print(sys.version_info[1])"
 	integrationVersionScript = `
-try:
-	from datadog_checks.%s import __version__
-	print(__version__)
-except ImportError:
-	pass
+import pkg_resources
+print(pkg_resources.get_distribution('%s')._version)
```
Comment (therve) [inline cmd/agent/app/integrations.go]: Why not `.version`?

File: releasenotes/notes/Use-pkg_resources-to-get-the-version-of-integrations-0af058b8fb39c548.yaml [hunk 5]
```diff
@@ -0,0 +1,4 @@
+fixes:
+  - |
+    Use 'pkg_resources' to collect the version of the integrations
```
Comment (ofek) [inline releasenotes/notes/Use-pkg_resources-to-get-the-version-of-integrations-0af058b8fb39c548.yaml]:
```suggestion
    Use ``pkg_resources`` to collect the version of the integrations
```

**Review by ofek — DISMISSED** [review]

**Review by therve — DISMISSED** [review]

File: cmd/agent/app/integrations.go [hunk 6]
```diff
@@ -690,9 +691,9 @@ func installedVersion(integration string) (*semver.Version, bool, error) {
 		return nil, false, nil
 	}
 
-	version, err := semver.NewVersion(outputStr)
+	version, err := semver.NewVersion(string(outputStr))
```
Comment (AlexandreYang) [inline cmd/agent/app/integrations.go]: (nit) Why the cast is needed here ?

**Review by AlexandreYang — DISMISSED** [review]
LGTM.

A minor comment.

File: cmd/agent/app/integrations.go (hunk 6 above)
Comment (FlorianVeaux) [reply cmd/agent/app/integrations.go]: It was not :)

**Review by AlexandreYang — APPROVED** [review]

**Review by albertvaka — APPROVED** [review]

**Pull request merged**